mailpoet/assets/css/src/components-automation-listing/_sections.scss

```diff
@@ -0,0 +1,171 @@
+:root {
+  --menu-width: 0;
+
+  @media screen and (min-width: 783px) {
+    --menu-width: 36px;
+  }
+
+  @media screen and (min-width: 961px) {
+    --menu-width: 160px;
+  }
+}
+
+@mixin full-width {
+  margin-left: calc(-50vw + var(--menu-width) / 2 + 50%);
+  padding-left: 104px;
+  padding-right: 104px;
+  width: calc(100vw - var(--menu-width));
+}
+
+.mailpoet-automation-section {
+  h2 {
+    font-size: 23px;
+    font-weight: 400;
+    line-height: 32px;
+    margin: 0;
+    padding: 0 0 8px;
+  }
+
+  p {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    margin: 0;
+    padding: 0 0 40px;
+  }
+}
+
+.mailpoet-automation-section-hero {
+  background: #fff;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-top: -20px;
+  padding: 65px;
+
+  @include full-width;
+
+  .mailpoet-automation-preheading {
+    display: block;
+    font-size: 11px;
+    letter-spacing: .2px;
+    line-height: 16px;
+    margin-bottom: 65px;
+    text-transform: uppercase;
+  }
+
+  h1 {
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 40px;
+  }
+
+  p {
+    font-size: 14px;
+    line-height: 22px;
+    margin-bottom: 32px;
+  }
+
+  > div {
+    width: 400px;
+  }
+}
+
+.mailpoet-section-templates {
+  padding: 48px 84px;
+
+  .mailpoet-section-template-list {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 40px;
+
+    li {
+      max-width: 336px;
+
+      button {
+        background: #fff;
+        border: 1px solid #dcdcde;
+        border-radius: 0;
+        color: #1d2327;
+        cursor: pointer;
+        height: auto;
+        padding: 24px;
+        text-align: left;
+
+        h3 {
+          font-size: 16px;
+          font-weight: 600;
+          line-height: 24px;
+        }
+      }
+    }
+  }
+
+  .components-button {
+    display: block;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 25px;
+    text-align: center;
+    text-underline-offset: 5px;
+  }
+}
+
+.mailpoet-section-build-your-own {
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 64px;
+
+  @include full-width;
+
+  ol {
+    list-style: decimal-leading-zero;
+    margin: 0;
+    max-width: 373px;
+    padding: 0;
+
+    .mailpoet-section-build-list-button {
```

Comment:
I see you're using a lot of nesting. It can become a bit hard to read, I'd say. We don't have any strict rules here, but I think we tend to put all unique-enough "FQN" qualifiers (like custom classes, e.g. `.mailpoet-section-build-list-button`) to the top level.

Reply:
Addressed in [7345003](https://github.com/mailpoet/mailpoet/pull/4461/commits/7345003b7bccf30ca7e17b1cfb9643b6ddac7450).

I actually like it "a bit nested". It groups the components CSS together, but can be too much, I agree. Will try to follow the teams standards.

Reply:
After rebase [d593be6](https://github.com/mailpoet/mailpoet/pull/4461/commits/d593be642f7b483a012a9bbb472b0941cca1181c)